public_suffix_list.dat

```diff
@@ -13570,8 +13570,8 @@ fly.dev
 shw.io
 edgeapp.net
 
-// Forgerock : https://www.forgerock.com
-// Submitted by Roderick Parr <roderick.parr@forgerock.com>
+// Ping Identity : https://www.pingidentity.com
```

Comment:
I see the sorting issue and formatting challenge.... P is later in the alphabet so the change needs to be moved to an appropriate place lower in the file